tests/Kiota.Builder.IntegrationTests/GenerateSample.cs

```diff
@@ -11,21 +11,21 @@ public class GenerateSample
         [InlineData(GenerationLanguage.Java, null)]
         [InlineData(GenerationLanguage.TypeScript, null)]
         [InlineData(GenerationLanguage.Ruby, null)]
-        [InlineData(GenerationLanguage.CSharp, "Microsoft.Kiota.Abstractions.Store.InMemoryBackingStore")]
-        [InlineData(GenerationLanguage.Java, "com.microsoft.kiota.store.InMemoryBackingStore")]
-        [InlineData(GenerationLanguage.TypeScript, "@microsoft/kiota-abstractions.InMemoryBackingStore")]
+        [InlineData(GenerationLanguage.CSharp, true)]
```

Comment:
We should probably change the nulls values in the lines above to false as I don't think we can really assign null to boolean.

Reply:
good catch, thanks, fixed.